lib/cli-engine/cli-engine.js

```diff
@@ -278,7 +278,8 @@ function verifyText({
  */
 function createIgnoreResult(filePath, baseDir) {
     let message;
-    const isHidden = /^\./u.test(path.basename(filePath));
+    const isHidden = filePath.split(path.sep)
+        .find(directory => /^\./u.test(directory));
```

Comment:
```suggestion
        .find(segment => /^\./u.test(segment));
```

Reply:
Small suggestion for clarity, since the final segment can be a directory or file.

Reply:
Done!